tableau-databricks/connection-builder.js

```diff
@@ -72,7 +72,7 @@ limitations under the License.
 	params["AsyncExecPollInterval"] = "1";
 
 	// Tell the ODBC driver that it is Tableau connecting.
-	params["UserAgentEntry"] = "Tableau";
+	params["UserAgentEntry"] = connectionHelper.GetProductName() + "_" + connectionHelper.GetProductVersion();
```

Comment:
The HTTP standard specifies the format as `Agent-Name/1.2.3 (some comment)`
It might also require spaces to be replaced

Reply:
GetProductName() should return TableauDesktop, TableauServer, or TableauPrep, so it should be enough to replace "_" with "/".